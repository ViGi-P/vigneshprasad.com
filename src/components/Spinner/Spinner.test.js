import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { Spinner } from './'

it('renders spinner', () => {
  const wrapper = shallow(<Spinner/>)
  expect(toJson(wrapper)).toMatchSnapshot()
})
