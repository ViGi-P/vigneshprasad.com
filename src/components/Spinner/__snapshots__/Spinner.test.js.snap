// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders spinner 1`] = `
<div
  className="spinner-container"
>
  <div
    className="spinner"
  >
    <svg
      viewBox="0 0 66 66"
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        className="length"
        cx="33"
        cy="33"
        fill="none"
        r="28"
        strokeLinecap="round"
        strokeWidth="8"
      />
    </svg>
    <svg
      viewBox="0 0 66 66"
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        cx="33"
        cy="33"
        fill="none"
        r="28"
        strokeLinecap="round"
        strokeWidth="8"
      />
    </svg>
    <svg
      viewBox="0 0 66 66"
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        cx="33"
        cy="33"
        fill="none"
        r="28"
        strokeLinecap="round"
        strokeWidth="8"
      />
    </svg>
    <svg
      viewBox="0 0 66 66"
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        cx="33"
        cy="33"
        fill="none"
        r="28"
        strokeLinecap="round"
        strokeWidth="8"
      />
    </svg>
  </div>
</div>
`;
